server/journeys/screen/confirmation/view.njk

```diff
@@ -0,0 +1,19 @@
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Confirmation - Screen CSIP referral" %}
+
+{% set theHtml = ("<div><strong>Decision: </strong>" + outcomeTypeDescription | escape + "</div>") %}
+
+{% block innerContent %}
+    {{ govukPanel({
+        titleText: "CSIP referral screening outcome recorded",
+        html: theHtml
+    }) }}
+    <p class="govuk-body">We've updated the status of the referral to "{{outcomeStatus}}".</p>
```

Comment:
it's
```
We’ve updated the status of the referral to “{{outcomeStatus}}”. 
```

not
```
We've updated the status of the referral to "{{outcomeStatus}}".
```

Reply:
`{{outcomeStatus | sentenceCase(false)}}`

Reply:
I am very blind to the differences with those unicode characters. Updating the tests took far longer than they should have lol

Reply:
i just avoid typing and copy and paste lol